### PR TITLE
Bugfixes: Drones, Infrared emitters, Implant cases and Limbs

### PR DESCRIPTION
**Pull request by inselc**
This PR contains bugfixes for:
- Drones being unable to matter-decompile burnt matches (#717). Burnt matches will be decompiled into a small amount of wood.
- Ghosts triggering infrared emitters (#644). Added a check whether the movable passing through the laser beam is actually "visible".
- Reagent transfer messages showing up when clicking on chem implant cases, even when the syringe is empty (#704). Added a check whether reagents have actually been transferred. The displayed message will now also display the actual amount transferred. Removed spawn(5)-delay to prevent click-stacking.
- Limb relocation messages not showing up for bystanders (#778).

### CODE REVIEW

**Comment by printer16** [comment]
"Ghosts triggering infrared emitters (#644). Added a check whether the movable passing through the laser beam is actually "visible"." Could you also toss this into the portable flash code? They suffer from the same issue.

**Comment by Lohikar** [comment]
I fixed that already.

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -1407,20 +1407,23 @@
 	var/obj/item/organ/external/current_limb = organs_by_name[choice]
 
 	if(self)
-		src << "<span class='warning'>You brace yourself to relocate your [current_limb.joint]...</span>"
+		U.visible_message("<span class='warning'>[U] tries to relocate their [current_limb.joint]...</span>", \
+		"<span class='warning'>You brace yourself to relocate your [current_limb.joint]...</span>")
 	else
-		U << "<span class='warning'>You begin to relocate [S]'s [current_limb.joint]...</span>"
+		U.visible_message("<span class='warning'>[U] tries to relocate [S]'s [current_limb.joint]...</span>", \
```
Comment (skull132) [inline code/modules/mob/living/carbon/human/human.dm]: For ref, as DM is compiled much like C, you don't need to break up single line statements with `\` unless dealing with literals or defines.

Comment (inselc) [inline code/modules/mob/living/carbon/human/human.dm]: Thanks for the heads-up! I'll keep this in mind.

**Pull request merged**